movo_common/movo_ros/bin/movo_config_eth

```diff
@@ -47,14 +47,14 @@ def validate_ipv4(argument,name):
     Each address must have 4 
     """
     if len(argument.split(".")) != 4:
-        print("Invalid %s, must be of the form xxx.yyy.zzz.www" % name)
+        print(("Invalid %s, must be of the form xxx.yyy.zzz.www" % name))
         exit(-1)
     for x in argument.split("."):
         if len(x) > 3:
-            print("Invalid %s, must be of the form xxx.yyy.zzz.www" % name)
+            print(("Invalid %s, must be of the form xxx.yyy.zzz.www" % name))
```

Comment:
- [ ] todo fix these

Reply:
Do we want to revert these changes back? These are proactive, redundant changes made by the 2to3 script using the flag to update print statements. Either these extra brackets need to be removed manually after running 2to3, or all the print statements using Python 2 syntax need to be changed manually without running 2to3.